cpp/include/cudf/io/parquet.hpp

```diff
@@ -389,9 +389,11 @@ class column_in_metadata {
   bool _use_int96_timestamp = false;
   // bool _output_as_binary = false;
   thrust::optional<uint8_t> _decimal_precision;
-  std::vector<column_in_metadata> children;
 
  public:
+
+  std::vector<column_in_metadata> children;
```

Comment:
Let me try your suggestion, I thought I tried it last night but it was late

Reply:
The reason it works for devavret is because he is initializing the table_metadata using a table. We don't have a table so when I call child(0) it returns children[0] which blows up basically I need to access the vector to call resize on it so it allocates the needed space

Reply:
I will remove the `[skip ci]` once you approve that this is valid

Reply:
This was requested because `table_input_metadata` needs a `table_view` to construct the default metadata from, in the same hierarchy as the `table_view`.  @razajafri pointed out that a table is not available during the time of construction of metadata in Spark's use case.